docs/examples/sequence_processing/video/video_reader_simple_example.ipynb

```diff
@@ -49,7 +49,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We need some video containers to process. We can use [Sintel](https://en.wikipedia.org/wiki/Sintel) trailer, which is a mp4 video container containing a h264 and under the Create Common license. Let's split it into 10s clips in order to check how `VideoReader` handles mutliple video files. This can be done easily with the `ffmpeg` standalone tool. Our prepared data can be found in [DALI_extra](https://github.com/NVIDIA/DALI_extra) repository. Snippet below verifies, that you have defined `DALI_extra` path as environment variable."
+    "We need some video containers to process. We can use [Sintel](https://en.wikipedia.org/wiki/Sintel) trailer, which is a mp4 video container containing a h264 and under the Create Common license. Let's split it into 10s clips in order to check how `VideoReader` handles mutliple video files. This can be done easily with the `ffmpeg` standalone tool. `DALI_EXTRA_PATH` environment variable should point to the place where data from [DALI extra repository](https://github.com/NVIDIA/DALI_extra) is downloaded. Please make sure that the proper release tag is checked out. Snippet below verifies, that you have defined `DALI_extra` path as environment variable."
```

Comment:
```suggestion
    "We need some video containers to process. We can use [Sintel](https://en.wikipedia.org/wiki/Sintel) trailer, which is an mp4 container containing an h.264 video and distributed under the Create Common license. We've split it into 5 second clips and divided the clips into labelled groups. This can be done easily with the `ffmpeg` standalone tool. `DALI_EXTRA_PATH` environment variable should point to the place where data from [DALI extra repository](https://github.com/NVIDIA/DALI_extra) is downloaded. Please make sure that the proper release tag is checked out. The snippet below verifies, that you have defined `DALI_extra` path as an environment variable."
```
```

Reply:
Legacy. Fixed